src/Components/SmartComponents/CVEDetailsPage/CVEDetailTableTitle.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { Spinner, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { useGetSingleDevice } from '../../../Helpers/APIHelper';
+import messages from '../../../Messages';
+
+const CVEDetailTableTitle = () => {
+    const intl = useIntl();
+    const [isLoading, setLoading] = useState(true);
+    const [edgeDeviceExist, setEdgeDeviceExist] = useState(false);
+    const [totalSystems, setTotalSystems] = useState(0);
+    const getEdgeDevice = useGetSingleDevice();
+    useEffect(() => {
+        getEdgeDevice().then(edgeDevice => {
+            setEdgeDeviceExist(edgeDevice?.count === 1);
+            setTotalSystems(edgeDevice?.total || 0);
+            setLoading(false);
+        });
```

Comment:
Can we please add `catch` or second argument to `then` method here to handle request failing? The fallback could be something like

```
setEdgeDeviceExist(false)
setTotalSystems(0);
setLoading(false);
```